File: WebApp/Models/PH/PH_MAILSP_D.cs
using System;
using System.Collections.Generic;

namespace WebApp.Models
{
    public class PH_MAILSP_D : JCLib.Mvc.BaseModel
    {
        public string INID { get; set; }
        public string MSGRECNO { get; set; }
        public string MSGNO { get; set; }
        public string AGEN_NO { get; set; }
        public string M_CONTID { get; set; }
        public string CREATE_TIME { get; set; }
        public string CREATE_USER { get; set; }
        public string UPDATE_TIME { get; set; }
        public string UPDATE_USER { get; set; }
        public string UPDATE_IP { get; set; }
        public string AGEN_NAMEC { get; set; }
        public string CONTRACNO { get; set; }
        
    }
}

File: WebAppVen/Scripts/app/model/PhMailspD.js
Ext.define('WEBAPP.model.PhMailspD', {
    extend: 'Ext.data.Model',
    fields: [
        { name: 'MGROUP', type: 'string' },
        { name: 'MSGNO', type: 'string' },
        { name: 'AGEN_NO', type: 'string' },
        { name: 'M_CONTID', type: 'string' },
        { name: 'CREATE_TIME', type: 'string' },
        { name: 'CREATE_USER', type: 'string' },
        { name: 'UPDATE_TIME', type: 'string' },
        { name: 'UPDATE_USER', type: 'string' },
        { name: 'UPDATE_IP', type: 'string' }
    ]
});